cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -319,6 +319,110 @@ def parse_harvard_opinions(reporter, volume):
             replace("Docket No.", ""). \
             replace("Docket Nos.", "")
         with transaction.atomic():
```

Comment:
Next time we do something like this, we should see how hard it is to do:

```
d = Docket(var=foo, zoo=boo)
cl = Cluster(asdf=asdf, asdf=asdf)
op = Opinion(blah, blah)
with transaction.atomic():
    d.save()
    cl.save()
    op.save()
```

I don't want to bother with this now, but indenting 100 lines to accomplish this feels icky.